src/components/v5/common/ActionSidebar/partials/forms/ManageTokensForm/hooks.ts

```diff
@@ -29,17 +29,24 @@ export const useManageTokens = (
     [colony?.tokens?.items],
   );
 
+  const blockchainNativeTokenAddress = colonyTokens.find(
+    ({ colonyTokensId }) => colonyTokensId === 'DEFAULT_TOKEN_ID',
+  )?.token.tokenAddress;
+
+  const colonyNativeTokenAddress = colony?.nativeToken.tokenAddress;
+
   const shouldShowMenu = useCallback(
```

Comment:
This should not be a callback either. See my rationale about this in the comments above.